requirements.txt

```diff
@@ -37,8 +37,11 @@ SpeechRecognition==3.1.3
 telepot==4.1
 
 # Text To Speech Module
-pygame==1.9.2b6
+pygame>=1.9.2b6
 pyvona==1.1
 
 # Creating Sphinx Language Model
-poster==0.8.1
\ No newline at end of file
+poster==0.8.1
+
+# Weather
```

Comment:
This shouldn't be here. This method creates problems in installation in many distros.